xcom2-launcher/xcom2-launcher/Forms/SettingsDialog.cs

```diff
@@ -119,13 +122,15 @@ private void bOK_Click(object sender, EventArgs e)
             Settings.NeverImportTags = neverAdoptTagsAndCatFromprofile.Checked;
             Settings.ShowQuickLaunchArguments = ShowQuickLaunchArgumentsToggle.Checked;
             Settings.CheckForPreReleaseUpdates = checkForPreReleaseUpdates.Checked;
+            Properties.Settings.Default.IsSentryEnabled = useSentry.Checked;
```

Comment:
Where is this checked/read?

Reply:
On application start in Program.InitSentry().

Reply:
Lol, it's first line -,-